src/npem.c

```diff
@@ -313,7 +313,7 @@ status_t npem_get_slot(char *device, char *slot_path, struct slot_response *slot
 
 	reg = read_npem_register(pdev, PCI_NPEM_CTRL_REG);
 	slot_res->state = npem_capability_to_ibpi(reg);
-	snprintf(slot_res->slot, PATH_MAX, "%s", path);
+	snprintf(slot_res->slot, PATH_MAX, "%s", basename(path));
```

Comment:
it seems to be risky, or it is already done.

Reply:
The generic print function was calling basename on the slot.  This PR removes that, thus necessitating the basename here to ensure same output.  Pci slot is already calling basename to assign value to the slot.

The reason I removed the basename in the print function is so that I each implementation of the get/set slot functions that fill out the slot structure are free to define what their slot id looks like.

Reply:
Thanks for clarification.
LGTM